plugins/woocommerce/src/Admin/API/AI/Middleware.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\API\AI;
+
+use Automattic\WooCommerce\StoreApi\Exceptions\RouteException;
+use WP_Error;
+
+/**
+ * Middleware class.
+ *
+ * @internal
+ */
+class Middleware {
```

Comment:
This is just a copy of this class https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/StoreApi/Routes/V1/AI/Middleware.php, the original one will be removed once all endpoints and migrated.